build.gradle

```diff
@@ -487,20 +487,20 @@ static URL availableBuildScriptUrl() {
     new URL("https://raw.githubusercontent.com/SinTh0r4s/ExampleMod1.7.10/main/build.gradle")
 }
 
-static boolean updateBuildScript() {
-    if (isNewBuildScriptVersionAvailable()) {
-        def buildscriptFile = new File("build.gradle")
+static boolean performBuildScriptUpdate(String projectDir) {
+    if (isNewBuildScriptVersionAvailable(projectDir)) {
+        def buildscriptFile = new File(projectDir, "build.gradle")
         availableBuildScriptUrl().withInputStream { i -> buildscriptFile.withOutputStream { it << i } }
         print("Build script updated. Please REIMPORT the project or RESTART your IDE!")
         return true
     }
     return false
 }
 
-static boolean isNewBuildScriptVersionAvailable() {
+static boolean isNewBuildScriptVersionAvailable(String projectDir) {
     Map parameters = ["connectTimeout": 2000, "readTimeout": 2000]
 
-    String currentBuildScript = new File("build.gradle").getText()
+    String currentBuildScript = new File(projectDir, "build.gradle").getText()
```

Comment:
what do you think about creating function which would do this for us, like `def projectFile(String name) = new File(projectDir, name)` or smth like that?

Reply:
Certainly a good thing! But is `projectDir` available in a function? Gonna test later